scripts/moneat-agent/moneat_agent/cursor_agent.py

```
@@ -0,0 +1,143 @@
+
+    log_fn(f"{label}...")
+
+    cmd = _build_cmd(
+        model=model,
+        workspace=workspace,
+        prompt=prompt,
+        mode=mode,
+        force=force,
+    )
+
+    proc = subprocess.Popen(
```

Ruff (0.15.6)

[error] `subprocess` call: check for execution of untrusted input

(S603)